tools/vfat-resize.c

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright (C) 2020  Red Hat, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Author: Vojtech Trefny <vtrefny@redhat.com>
+ *
+ * This is just a small helper program for (V)FAT filesystem resizing using libparted.
+ * We don't want this to be part of the library because libparted is licensed under GPLv3
+ * and we want to keep libblockdev LGPL compatible.
+ *
+ */
+
+#include <stdlib.h>
+#include <stdio.h>
+#include <unistd.h>
+#include <string.h>
+#include <glib.h>
+#include <parted/parted.h>
+#include <bytesize/bs_size.h>
+
+
+void print_usage (const char *cmd) {
+    g_print ("Usage: %s device [size]\n"
```

Comment:
This is from `<glib/gprintf.h>` or something like that, isn't it?

Reply:
No, it's from `glib/gmessages.h` which is included in `glib.h`. You probably think of `g_vasprintf` which needs `glib/gprintf.h` included explicitly.